python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1370,6 +1381,7 @@ def test_delta_byte_array_roundtrip(
     nrows, add_nulls, max_string_length, str_encoding, tmpdir
 ):
     null_frequency = 0.25 if add_nulls else 0
+    prefer_dba = str_encoding == "DELTA_BYTE_ARRAY"
```

Comment:
I'd prefer the full name everywhere in this PR. `prefer_dba` -> `prefer_delta_byte_array` throughout.
```suggestion
    prefer_delta_byte_array = str_encoding == "DELTA_BYTE_ARRAY"
```